src/WebJobs.Script/Rpc/LanguageWorkerConstants.cs

```diff
@@ -46,5 +46,8 @@ public static class LanguageWorkerConstants
         public const string RpcHttpCookies = "cookies";
         public const string RpcHttpStatusCode = "statusCode";
         public const string RpcHttpStatus = "status";
+
+        // Capabilities
+        public const string RawHttpBodyBytes = "RawHttpBodyBytes";
```

Comment:
Remove this and send it as part of the follow up PR that handles RawHttpBodyBytes